src/h5web/nexus-visualizer/NexusVisualizer.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactElement } from 'react';
+import type { HDF5Group } from '../providers/models';
+import styles from '../Visualizer.module.css';
+
+interface Props {
+  group: HDF5Group;
+}
+
+function NexusVisualizer(props: Props): ReactElement {
+  const { group } = props;
+  return (
+    <div className={styles.visualizer}>
+      <div className={styles.visBar} />
+      <p className={styles.fallback}>NeXus group {group.id}</p>
```

Comment:
Just showing the ID of the group for now.

This component may evolve to become a lot similar to `DatasetVisualizer` (a `VisSelector`, a toolbar, a `VIS_DEFS` object, etc.), but until we know more, I think it's better to duplicate things as needed instead of adding complexity to the existing code that may lead us to a dead end.

Reply:
Good call :+1: